.gitignore

```diff
@@ -63,4 +63,11 @@ node_modules
 
 /stash
 dist
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
This commit seems unrelated to your change. Excluding the root the stash directory should have been all that was necessary.

Reply:
yeah, i kinda didnt want to push my changes to the .gitignore in the first place. ill revert the changes for this pr ;)